ts/parts/Navigator.ts

```diff
@@ -505,6 +505,15 @@ extend(defaultOptions, {
              * The type of the navigator series. Defaults to `areaspline` if
              * defined, otherwise `line`.
              *
+             * Heads up:
```

Comment:
Could we handle this with smarter default options so that we don't need to do manual warnings?

Reply:
Sure, not a problem